octopoes/bits/missing_dkim/missing_dkim.py

```diff
@@ -17,12 +17,11 @@ def run(
         not tldextract.extract(input_ooi.name).subdomain
         # don't report on findings on tlds
         and tldextract.extract(input_ooi.name).domain
-    ):
-        if not additional_oois:
-            ft = KATFindingType(id="KAT-NO-DKIM")
-            yield ft
-            yield Finding(
-                ooi=input_ooi.reference,
-                finding_type=ft.reference,
-                description="This hostname does not support DKIM records",
-            )
+    ) and not additional_oois:
```

Comment:
There isn't a reason to put the parentheses only around the first two terms, I think moving `and not additional_oois` inside makes it clearer, maybe also add a comment like `# only report finding when there is no DKIM record`. The same for DMARC and SPF bits.

Reply:
The parentheses are a format thing because of multiline stuff. Good point about the comment